youtube_dl/extractor/brightcove.py

```diff
@@ -508,7 +510,21 @@ def _extract_urls(webpage):
                 'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
                 % (account_id, player_id, embed, video_id))
 
-        return entries
+        # 	<video data-brightcove-video-id="5320421710001" data-account="245991542" data-player="SJWAiyYWg" data-embed="default" class="video-js" controls itemscope itemtype="http://schema.org/VideoObject">
+        for video in re.findall(r'(?i)(<video[^>]+>)', webpage):
+            attrs = extract_attributes(video)
+                
+            video_id    = dict_get(attrs, ['data-brightcove-video-id', 'data-video-id'])
+            account_id  = attrs.get('data-account')
+            player_id   = attrs.get('data-player')
+            embed       = attrs.get('data-embed')
+
+            if video_id and account_id and player_id and embed:
+                entries.append(
+                    'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
+                    % (account_id, player_id, embed, video_id))
+
+	return entries
```

Comment:
Seems a waste, but how should I do it. I asked above:

> I'm also not sure if you mean I should have a `BostonGlobeIE(BrightcoveNewIE)` in `bostonglobe.py` and bypass the generic extractor (I guess this could handle the `video_id` problem?), or a `BrightcoveAltVideoTagIE(BrightcoveNewIE)` in `brightcove.py` that handles this special case of brightcove with the `data-brightcove-video-id` instead of `data-video-id`?

Thanks.